e2e/compatibility_test.go

```diff
@@ -0,0 +1,102 @@
+
+	icmd.RunCmd(icmd.Command("wget", "-q", "-O", tmpDir.Join("img.tar"), url)).Assert(t, icmd.Success)
+
+	cmd.Command = dockerCli.Command("load", "-i", tmpDir.Join("img.tar"))
+	result := icmd.RunCmd(cmd).Assert(t, icmd.Success)
+	var digest string
+	for _, line := range strings.Split(result.Combined(), "\n") {
+		if strings.Contains(line, "sha256:") {
+			digest = strings.Split(line, "sha256:")[1]
+		}
+	}
+	digest = strings.Trim(digest, " \r\n")
+
+	cmd.Command = dockerCli.Command("tag", digest, tag)
+	icmd.RunCmd(cmd).Assert(t, icmd.Success)
+
+	return "sha256:" + digest
+}
+
+func getBundle(t *testing.T, url string) []byte {
+	client := http.Client{Timeout: time.Minute * 1}
+	res, err := client.Get(url)
+	assert.NilError(t, err)
+
+	body, err := ioutil.ReadAll(res.Body)
+	assert.NilError(t, err)
+
+	return body
+}
+func TestBackwardsCompatibilityV1(t *testing.T) {
+	runWithDindSwarmAndRegistry(t, func(info dindSwarmAndRegistryInfo) {
+
+		cmd := info.configuredCmd
+		appName := "app-e2e"
+		tmpDir := fs.NewDir(t, appName)
+		//defer tmpDir.Remove()
+
+		bundle := getBundle(t, "https://github.com/docker/app-e2e/raw/master/bundles/v0.9.0/bundle.json")
+
+		invocDigest := importAndTagImage(t, cmd, tmpDir, "app-e2e:0.1.0-invoc", "https://github.com/docker/app-e2e/raw/master/images/v0.9.0/app-e2e-invoc.tar")
+		backendDisgest := importAndTagImage(t, cmd, tmpDir, "app-e2e/backend", "https://github.com/docker/app-e2e/raw/master/images/v0.9.0/backend.tar")
+		frontendDigest := importAndTagImage(t, cmd, tmpDir, "app-e2e/frontend", "https://github.com/docker/app-e2e/raw/master/images/v0.9.0/frontend.tar")
+
+		bundle = bytes.ReplaceAll(bundle, []byte("<digest-app-e2e-invoc>"), []byte(invocDigest))
+		bundle = bytes.ReplaceAll(bundle, []byte("<digest-backend>"), []byte(backendDisgest))
+		bundle = bytes.ReplaceAll(bundle, []byte("<digest-frontend>"), []byte(frontendDigest))
+
+		err := ioutil.WriteFile(tmpDir.Join("bundle.json"), bundle, 0644)
+		assert.NilError(t, err)
+
+		fmt.Println(tmpDir.Join("bundle.json"))
+		// Install a Docker Application Package with an existing failed installation is fine
+		cmd.Command = dockerCli.Command("app", "run", "--cnab-bundle-json", tmpDir.Join("bundle.json"), "--name", appName)
+		checkContains(t, icmd.RunCmd(cmd).Assert(t, icmd.Success).Combined(),
+			[]string{
+				fmt.Sprintf("Creating service %s_backend", appName),
+				fmt.Sprintf("Creating service %s_frontend", appName),
+				fmt.Sprintf("Creating network %s_default", appName),
+			})
+		// List the installed application
+		cmd.Command = dockerCli.Command("app", "ls")
+		checkContains(t, icmd.RunCmd(cmd).Assert(t, icmd.Success).Combined(),
+			[]string{
+				`RUNNING APP\s+APP NAME\s+LAST ACTION\s+RESULT\s+CREATED\s+MODIFIED\s+REFERENCE`,
+				fmt.Sprintf(`%s\s+%s \(0.1.0\)\s+install\s+success\s+.+second[s]?\sago\s+.+second[s]?\sago\s+`, appName, appName),
+			})
+
```

Comment:
I think we are missing some actions implemented by the bundle, like `Render`, `inspect` and `status`. I think the purpose of this test is to check that every action can be called.